Contracts/KeyPair.cs

```diff
@@ -5,10 +5,10 @@
     /// </summary>
     public class KeyPair
     {
-        public string PrivateKey { get; set; }
```

Comment:
It's a bit wired that the private key is string and the pubic is byte array... I think you should choose either one for both of them and use the same type. if you get String here, you can internally transform to byte array. otherwise it looks weird...